### ISSUE TITLE
v5.15.0 NFO / Artwork errors

### ISSUE THREAD

**Issue by ForeheadFox** [Inrixia/Floatplane-Downloader#216]
Quote: "getting these errors when starting the windows exe now after the update.
Seems to still download episodes but does not grab the artwork and Nfo file anymore."

![image](https://github.com/user-attachments/assets/c140c6d4-d64d-45f3-93b6-88785373de41)
![image](https://github.com/user-attachments/assets/4fa8844d-7608-4d88-bf60-782502e6ebbf)
![image](https://github.com/user-attachments/assets/545f41ff-b18a-4e41-aefd-b997b492a9e5)

Deleting the db folder and rebuilding does nothing.

**Comment by tobias-theobald** [comment]
I have the same issue (x86-64 linux docker) and some additional logs that may help:

```
2024-10-27 00:45:14.845175+02:00[10/26/2024, 10:45:14 PM] My RGB is better than yours again - Govee Gaming Sync Box 2.0 (SPONSORED) - Waiting on delivery cdn...
...
2024-10-27 00:48:58.883272+02:00[10/26/2024, 10:48:58 PM] My RGB is better than yours again - Govee Gaming Sync Box 2.0 (SPONSORED) - Download started!
...
2024-10-27 00:49:49.177003+02:00[10/26/2024, 10:49:49 PM] My RGB is better than yours again - Govee Gaming Sync Box 2.0 (SPONSORED) - Download complete!
2024-10-27 00:49:49.177036+02:00[10/26/2024, 10:49:49 PM] My RGB is better than yours again - Govee Gaming Sync Box 2.0 (SPONSORED) - Saving .nfo
2024-10-27 00:49:49.180672+02:00[10/26/2024, 10:49:49 PM] My RGB is better than yours again - Govee Gaming Sync Box 2.0 (SPONSORED) - An error occoured: The operation would yield an incorrect node tree. Document already has a document element node. Node is title. - Retrying in 1s [1/5]
2024-10-27 00:49:50.181856+02:00[10/26/2024, 10:49:50 PM] My RGB is better than yours again - Govee Gaming Sync Box 2.0 (SPONSORED) - Muxing ffmpeg metadata...
2024-10-27 00:49:50.577943+02:00[10/26/2024, 10:49:50 PM] My RGB is better than yours again - Govee Gaming Sync Box 2.0 (SPONSORED) - Download & Muxing complete!
```

The relevant line here is probably

```
An error occoured: The operation would yield an incorrect node tree. Document already has a document element node. Node is title.
```

I just set the floatplane downloader up yesterday and the error is happening for every download as far as I can tell.

Appreciate the project!

**Comment by tobias-theobald** [comment]
I think I may have figured it out: You recently switched to `xmlbuilder2`, which has a subtly different API on the `create` function. This change is not accounted for in this project's code.

Current code from `src/lib/Video.ts`:

```
		const nfo = builder
			.create("episodedetails")
			.ele("title")
			.txt(this.videoTitle)
			.up()
...
```

I've isolated the code and was able to replicate the issue. Passing a string to `create` like this will cause the string to be interpreted as xml to be parsed. Anyway, fixing it is as easy as creating the `episodedetails` node in a separate command:

```
		const nfo = builder
			.create()
			.ele("episodedetails")
			.ele("title")
			.txt(this.videoTitle)
			.up()
...
```

The artwork download then fails because nfo creation throwing an error will cause the `download` function at line 143 to then go into the catch at line 170 then the retry logic will continue with remuxing, never retrying the artwork download of nfo creation.

I have tested this fix in isolation but want to still test if it works in the context of the whole project, so PR should be incoming later today.
